src/Components/UI/Navigation/VerticalNavBar.css

```diff
@@ -0,0 +1,106 @@
+.vertical-nav-container {
+  position: sticky;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  top: 4rem;
+  min-width: 15rem;
+  height: 100%;
+  padding: 0 0 1rem 0.3rem;
+}
+
+.nav-bar-mobile {
+  display: none;
+}
+
+.vertical-nav-menu,
+.vertical-nav-category {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 0;
+}
+
+.vertical-nav-menu h1,
+.vertical-nav-category h1 {
+  font-size: 0.95rem;
+  padding: 5px 8px;
+  margin-left: 8px;
+  margin-bottom: 5px;
+  color: var(--title-color);
+}
+
+.vertical-nav-menu a > div,
+.vertical-nav-category a > div {
```

Comment:
could have used is() for this

Reply:
I guess is:() is used for selecting elements, because it's giving me the error and does not work for me, as I am using classes.